test/utils.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const tape = require('tape');
+
+module.exports = {
+  test: (description, body) => {
+    tape(description, (t) => {
+      t.deepStrictEqual = t.deepEqual;
+      global.assert = t;
+      body();
```

Comment:
in tape, the callback can be async, so should this perhaps be `await body();`?

Reply:
since, as of right now, we're a fully sync API, I'd rather keep it simple.